x-pack/test/observability_ai_assistant_functional/tests/conversations/index.spec.ts

```diff
@@ -417,30 +417,13 @@ export default function ApiTest({ getService, getPageObjects }: FtrProviderConte
 
                   expect(events.length).to.eql(1);
 
-                  const { messageWithFeedback, conversation } = events[0]
+                  const { feedback, conversation } = events[0]
                     .properties as unknown as ChatFeedback;
 
-                  expect(messageWithFeedback.feedback).to.eql('positive');
-                  expect(messageWithFeedback.message.message).to.eql({
-                    content: 'My response',
-                    function_call: {
-                      arguments: '',
-                      name: '',
-                      trigger: 'assistant',
-                    },
-                    role: 'assistant',
-                  });
-
-                  expect(conversation.conversation.title).to.eql('My title');
```

Comment:
Shouldn't we be checking to make sure the title and other removed properties do not exist instead of removing the assertion altogether?